app/controllers/subscriber_lists_controller.rb

```diff
@@ -31,6 +31,24 @@ def create
     render json: subscriber_list.to_json
   end
 
+  def update
+    unless has_updatable_parameters?
+      render json: {
+        error: "Must include at least one of: #{updatable_parameters.join(', ')}",
+      }, status: :bad_request
+      return
+    end
+
+    subscriber_list = SubscriberList.find_by(slug: params[:slug])
+
+    unless subscriber_list
+      render json: { error: "Could not find the subscriber list" }, status: :not_found and return
```

Comment:
The `... and return` pattern is useful in the construct `do_thing and return if condition`, but if you're not using an inline condition I think it's better to split it across two lines.